src/jdk.httpserver/share/classes/module-info.java

```diff
@@ -37,6 +37,55 @@
  * <p>The {@link com.sun.net.httpserver.spi} package specifies a Service Provider
  * Interface (SPI) for locating HTTP server implementations based on the
  * {@code com.sun.net.httpserver} API.
+ * <p>
+ * <b id="httpserverprops">System properties used by the HTTP server API</b>
+ * <p>
+ * The following is a list of JDK specific system properties used by the default HTTP
+ * server implementation in the JDK.
+ * <ul>
+ * <li><p><b>{@systemProperty sun.net.httpserver.idleInterval}</b> (default: 30 sec)<br>
+ * Maximum duration in seconds which an idle connection is kept open. This timer
+ * has an implementation specific granularity that may mean that idle connections are
+ * closed later than the specified interval.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpserver.maxConnections}</b> (default: -1)<br>
+ * The maximum number of open connections at a time. This includes active and idle connections.
+ * If zero or negative, then no limit is enforced.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxIdleConnections}</b> (default: 200)<br>
+ * The maximum number of idle connections at a time.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.drainAmount}</b> (default: 64K)<br>
```

Comment:
Hello Michael, sorry I missed this in my previous review. The value for `drainAmount` is expected to be a `long` value. So I think the default value here should be `65536`. Readers might take the `64K` value literally and might attempt to pass similar values to this property which will then lead to failures.

Reply:
Do we additionally have to specify the behaviour when the value is `<=0`?

Reply:
Okay, I can change the `64K` to `65536`. I think the spec already covers the `<=0` case. Thanks!